### PR TITLE
macOS 10.13 Week 40 Software Updates

### PR DESCRIPTION
**Pull request by Kutkovsky**
Software updates:
- java 13 : OpenJDK Runtime Environment Zulu13.27+9-CA (build 13+33)

Android:
- Android Emulator 29.2.1 

Xamarin:
- Visual Studio for Mac: 8.3.0.1805
- Mono Version: 6.4.0
- Xamarin.iOS SDK Version: 13.2.0.42
- Xamarin.Mac SDK Version: 5.16.1.24
- Xamarin.Mac SDK Version: 6.2.0.42
- Xamarin.Android SDK Version: 10.0.0.43

### CODE REVIEW

File: images/macos/macos-10.13-Readme.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
 
 ## Image Changes
 
-The following Java versions are available on the VM image: 7, 8, 11, 12.
+The following Java versions are available on the VM image: 7, 8, 11, 12, 13.
 	Java 9 and 10 were deprecated.
 
 Previously, Microsoft hosted Mac machines had JDKs pre-installed that were overloaded by complex licensing, end-user restrictions, and lack of long-term support. In this image change, we replaced the JDKs with tested, certified, LTS builds of OpenJDK from Azul Systems. Java developers can now build and run production Java applications using Azul Systems Zulu Enterprise builds of OpenJDK without incurring additional support costs. This new offering is designed to make Microsoft hosted Java builds and deployments worry-free by incorporating quarterly security updates and bug fixes as well as critical out-of-band updates and patches as needed.
```
Comment (maxim-lobanov) [inline images/macos/macos-10.13-Readme.md]: I suggest to remove Java switching from `Image Changes` section. This section is needed to notify customers about future changes and recent changes on image but switching to Azul JDK was done ~4 months ago

Comment (maxim-lobanov) [reply images/macos/macos-10.13-Readme.md]: The same for Mojave image
cc: @al-cheb

Comment (Kutkovsky) [reply images/macos/macos-10.13-Readme.md]: Fixed

File: images/macos/macos-10.13-Readme.md [hunk 2]
```diff
@@ -2,13 +2,6 @@
...
 The following software is installed on machines in the Azure Pipelines **macOS-10.13** VM image ('Hosted macOS High Sierra' pool).
 
-## Image Changes
-
-The following Java versions are available on the VM image: 7, 8, 11, 12.
-	Java 9 and 10 were deprecated.
-
-Previously, Microsoft hosted Mac machines had JDKs pre-installed that were overloaded by complex licensing, end-user restrictions, and lack of long-term support. In this image change, we replaced the JDKs with tested, certified, LTS builds of OpenJDK from Azul Systems. Java developers can now build and run production Java applications using Azul Systems Zulu Enterprise builds of OpenJDK without incurring additional support costs. This new offering is designed to make Microsoft hosted Java builds and deployments worry-free by incorporating quarterly security updates and bug fixes as well as critical out-of-band updates and patches as needed.
-
 #### Xcode 10.1 set by default
```
Comment (maxim-lobanov) [inline images/macos/macos-10.13-Readme.md]: It can be removed too

Comment (Kutkovsky) [reply images/macos/macos-10.13-Readme.md]: Yes, it is red (had been removed) already :)

**Pull request merged**